### DISCUSSION TITLE
Help with time settings and battery savings

### DISCUSSION THREAD

**Discussion by spencer245** [gopro/labs#1118] [Q&A] [answered]
I'm doing an outdoor timelapse and need my camera to only shoot monday thru friday but i can't seem to figure out a command. I would also like to get my camera to turn on and off between every 5 minute photo to save battery. Here's my qr code so far:

![QR-0700NmPr0fWw50aAaSb0d0i8M1MOWN](https://github.com/user-attachments/assets/bcb44aa9-84a2-4169-accb-c7d0ba62684b)

**Comment by dnewman-gpsw** [comment]
Don't combine actions (!) and settings in a single QR Code, at least until you know what you are doing.  

Learn from https://gopro.github.io/labs/control/construction/

**Reply by spencer245** [reply]
Copy. So I ditched my old code and ended up with 

mPdP>06:59<17:00!435SQ~!07:00S!1R

Now when i scan this qr code with my gopro, it says "Start time 07:13:00 (5 minutes) press start button to cancel"

Why, after clearly inputting the start time as 7 am, is it stating the above message? 

Thank you

**Reply by dnewman-gpsw** [reply, marked as answer]
That is the command states and is working correctly.  

```mPdP``` <- mode Photo , default Photo
```>06:59<17:00``` <-  if(time > 6:59am and time < 17:00)
```!435SQ``` <-  then sleep for 435s then Start
```~!07:00S``` <-  else sleep until 7am.
```!1R``` <- Repeat

Your code is working.  If you want to sleep until 7am, start it before 7am or after 5pm, otherwise the code assumes you want to capture today as ```if(time > 6:59am and time < 17:00)``` is true.
